week-8/Homework/mandatory/4-programmer-humour/scripts.js

```diff
@@ -0,0 +1,39 @@
+function showImg(param) {
+    let image = document.getElementById("image");
+    image.src = param
+}
+
+function getApiData() {
+
+    fetch('https://xkcd.now.sh/?comic=latest')
+        .then(response => {
+            console.log(response)
+            return response.json()
+        })
+        .then(data => {
+            console.log(data)
+            let resultMonth = data.month
```

Comment:
Can you talk through what all of these variable assignments and logs are for?

Reply:
thank you for checking. Sorry I forgot to delete log. It was for test my data object.